azure-kusto-data/azure/kusto/data/aio/request.py

```diff
@@ -0,0 +1,75 @@
+import io
+from collections import namedtuple
+from datetime import timedelta
+from typing import Union
+
+from .security import _AadHelper
+from .._decorators import documented_by, aio_documented_by
+from ..exceptions import KustoAioSyntaxError
+from ..request import KustoClient as KustoClientSync, _KustoClientBase, KustoConnectionStringBuilder, ClientRequestProperties
+from ..response import KustoResponseDataSet
+
+try:
+    from aiohttp import ClientResponse, ClientSession
+except ImportError:
+    raise KustoAioSyntaxError()
+
+ResponseTuple = namedtuple("ResponseTuple", ["response", "response_json"])
+
+
+@documented_by(KustoClientSync)
```

Comment:
please don't do this.
it blocks the ability to render this file to a readthedocs page. comments should be static